vocode/streaming/action/transfer_call.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Optional, Type
+from pydantic import BaseModel, Field
+from vocode.streaming.action.base_action import BaseAction
+from vocode.streaming.models.actions import ActionInput, ActionOutput
+from twilio.rest import Client as TwilioClient
+from vocode.streaming.telephony.config_manager.redis_config_manager import (
+    RedisConfigManager,
+)
+
+
+class TransferCallParameters(BaseModel):
+    pass
+
+
+class TransferCallResponse(BaseModel):
+    status: str = Field("success", description="status of the transfer")
+
+
+class TransferCall(BaseAction[TransferCallParameters, TransferCallResponse]):
+    description: str = "transfers the call. use when you need to connect the active call to another phone line."
+    action_type: str = "action_transfer_call"
+    parameters_type: Type[TransferCallParameters] = TransferCallParameters
+    response_type: Type[TransferCallResponse] = TransferCallResponse
+
+    def __init__(self, to_phone: str, **kwargs):
+        super().__init__(**kwargs)
+        self.to_phone = to_phone
+        # TODO: need to support all config managers
+        self.config_manager = RedisConfigManager()
```

Comment:
can we use the ConversationStateManager here?

or even better just use the TwilioPhoneCallAction 
https://github.com/vocodedev/vocode-python/blob/e4cd86b9bca32bb3d65b6e4e4b50caf948775d87/vocode/streaming/action/phone_call_action.py#L31

to grab the sid